website/sidebars.json

```diff
@@ -10,21 +10,26 @@
             "basic-concepts/state-tokens",
             "basic-concepts/node-mining"
         ],
+        "Tooling": [
+            "tooling/introduction",
+            "tooling/from-community"
```

Comment:
from-community is not very good title for this page, change it to "commnuity contribution" or another better one?

Reply:
yes, I prefer to "community contribution" will update it.